src/test/java/io/cdap/plugin/splunk/source/batch/SplunkBatchSourceConfigBuilder.java

```diff
@@ -67,6 +69,7 @@
   private String indexedLatestTime;
   private Long searchResultsCount;
   private String schema;
+  private boolean mockConnection = false;
```

Comment:
I think this hack is too dirty. Maybe it's better to move connection related validation to another method? Maybe even not in config class.

Reply:
Done.